website/blog/2024-09-01-model-as-code-logging/index.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Model-as-Code Logging in MLflow for Enhanced Model Management
+tags: [pyfunc, genai]
+slug: model_as_code
+authors: [awadelrahman-ahmed, daniel-liden]
+thumbnail: img/blog/thumbnail_model_as_code.gif
+---
+
+We all—well, most of us—remember November 2022 when the public release of ChatGPT by OpenAI marked a significant turning point in the world of AI. While GenAI had been evolving for some time, ChatGPT, built on OpenAI's GPT-3.5 architecture, quickly captured the public’s imagination. This led to an explosion of interest in GenAI, both within the tech industry and among the general public.
```

Comment:
The hyphen conversational abrupt stall threw me for a loop (I thought it was a typo). Perhaps encapsulate in parenthesis?